src/components/pages/index-page/IndexPage.tsx

```diff
@@ -1,13 +1,57 @@
 import React from 'react';
 import Link from 'next/link';
 import { Routes } from '../../../constants/Routes';
+import { GridCellKind } from "@glideapps/glide-data-grid";
```

Comment:
Using the `GridCellKind` enum outside of the "dynamic" causes an error when trying to build the Next.js app for production. During development you do see the error in the terminal but in the browser it is fine. 

`ReferenceError: document is not defined`


```
ReferenceError: document is not defined
at clearCacheOnLoad (file:///Users/robert/WebstormProjects/my-next.js-starter/node_modules/@glideapps/glide-data-grid/dist/js/index.js:927:3)
at file:///Users/robert/WebstormProjects/my-next.js-starter/node_modules/@glideapps/glide-data-grid/dist/js/index.js:934:6
  at ModuleJob.run (node:internal/modules/esm/module_job:185:25)
at async Promise.all (index 0)
at async ESMLoader.import (node:internal/modules/esm/loader:281:24)
at async importModuleDynamicallyWrapper (node:internal/vm/module:437:15)
info  - Collecting page data
  [=   ] info  - Generating static pages (3/11)file:///Users/robert/WebstormProjects/my-next.js-starter/node_modules/@glideapps/glide-data-grid/dist/js/index.js:927
  if (((_a = document == null ? void 0 : document.fonts) == null ? void 0 : _a.ready) === void 0)
    ^

```